src/repl/scala/tools/nsc/InterpreterLoop.scala

```diff
@@ -3,10 +3,15 @@ package scala.tools.nsc
 import interpreter._
 import java.io._
 
-/** A compatibility stub.
+/** A compatibility stub for sbt.
  */
-@deprecated("Use a class in the scala.tools.nsc.interpreter package.", "2.9.0")
+@deprecated("Use scala.tools.nsc.interpreter.ILoop.", "2.9.0")
 class InterpreterLoop(in0: Option[BufferedReader], out: PrintWriter) extends ILoop(in0, out) {
   def this(in0: BufferedReader, out: PrintWriter) = this(Some(in0), out)
   def this() = this(None, new PrintWriter(scala.Console.out))
+
+  override protected final val isSbt = true
```

Comment:
Given Adriaan's previous comment, at a guess I would think this should be `protected[nsc]` (or tools or scala).

nsc is principle of least power and scala is principle of maximal flexibility?